docs/semconv/database.md

```diff
@@ -17,17 +17,20 @@ This page tries to describe a semantic convention for the attributes and metrics
+| [`db.system`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-spans.md#connection-level-attributes) | string | An identifier for the database management system (DBMS) product being used.                                                                                      | db2, damengdb                | Required                                   |
+| `db.version`                                                                                                                                | string | The version of the database                                                                                                                                      | V11.5, V8                    | Required                                   |
+| `service.name`                                                                                                                              | string | This attribute is used to describe the entity name.                                                                                                              | damengdb@DAMENG              | Required                                   |
+| `service.instance.id`                                                                                                                       | string | This attribute is used to describe the entity ID of the current object and consists of server.address, server.port, and db.name.                                 | db.testdb.com:5236@DAMENG    | Conditionally Required: If applicable.     |
+| `db.entity.parent.id`                                                                                                                       | string | This attribute is used to describe the parent entity ID of the current object and consists of server.address, server.port and db.name or instance.name together. | db.testdb.com:5236@db2inst1  | Conditionally Required: If applicable.     |
+| `db.entity.type`                                                                                                                            | string | This attribute is used to describe the type of the current object.                                                                                               | DATABASE, INSTANCE           | Conditionally Required: If applicable.     |
+| `db.database.name`                                                                                                                          | string | Name of the Database.                                                                                                                                            | prod_db, instana             | Conditionally Required: If applicable.     |
```

Comment:
I think one informix instance could has more than one database in it. So the `db.database.name` should be considered as a DataPointAttribute for metrics which related to `db.database`.